x/feemarket/types/params.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 var (
+	// DefaultBaseFee is 0.1 for the ICS Evmos chain
+	DefaultBaseFee = math.LegacyNewDecWithPrec(100000000000, 2)
```

Comment:
nit: for readability
```suggestion
	DefaultBaseFee = math.LegacyNewDecWithPrec(100_000_000_000, 2)
```